runtime/interpreter/account.go

```diff
@@ -0,0 +1,219 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
```

Comment:
Moved from large `value.go` and refactored to use `SimpleCompositeValue`, to not have to pay for the overhead of atree ordered map in a normal (storable) composite value